docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -894,6 +895,15 @@ Ulimit options. Sets the ulimits values inside of the container.
 
 This key can be listed multiple times.
 
+### `Umask=`
+
+Set the umask of the process. This is equivalent to the Podman `--umask` and generally has the form `[UMASK]`
```

Comment:
Remove the "and generally has the form `[UMASK]`" part I don't see how this adds any value and what means form `[UMASK]`?